components/apimgt/org.wso2.carbon.apimgt.rest.api.common/src/main/java/org/wso2/carbon/apimgt/rest/api/common/utils/JWTUtil.java

```diff
@@ -101,6 +101,7 @@ public static boolean handleScopeValidation(HashMap<String,Object> message, Sign
                     carbonContext.setTenantDomain(tenantDomain);
                     carbonContext.setTenantId(tenantId);
                     carbonContext.setUsername(username);
+                    message.put("AUTH_TOKEN_INFO", oauthTokenInfo);
```

Comment:
Move this (`AUTH_TOKEN_INFO`) to the constants and use it form the other places.